src/highdicom/content.py

```diff
@@ -676,6 +717,12 @@ def __init__(
                 )
             self.append(item)
 
+    @property
+    def substances(self) -> List[CodedConcept]:
```

Comment:
This is not related to your changes here but I noticed that there's no check in the constructor that the passed list of substances is not empty,  which although I haven't checked the standard is a mistake I assume and should probably be fixed.

Reply:
You are right, there should be at least one substance. Addressed via cfa78c1779cc49e9e60ec382a220bd630714fdb7